config/initializers/exception_notification.rb

```diff
@@ -0,0 +1,14 @@
+require 'exception_notification/rails'
+require 'exception_notification/sidekiq'
+
+EXCEPTION_RECIPIENTS = JSON.parse(Figaro.env.exception_recipients).freeze
```

Comment:
This breaks if `Figaro.env.exception_recipients` is nil. Any thoughts on changing this up so that it works if `exception_recipients` is nil.

Reply:
This is currently a required Figaro key, so it can't be nil.